app/(afterLogin)/(timesheetInformation)/timesheet/settings/approvals/_component/editWorkFLow/index.tsx

```
@@ -73,8 +73,8 @@ const EditWorkFLow = () => {
         onError: (error: any) => {
           NotificationMessage.error({
             message: 'Error',
-            description:
-              error?.response?.data?.message ?? 'Something went wrong',
+            description: error?.response?.data?.message ?? 'Something went wrong',
+
```

ESLint

[error] Replace `·error?.response?.data?.message·??·'Something·went·wrong',⏎` with `⏎··············error?.response?.data?.message·??·'Something·went·wrong',`

(prettier/prettier)